cookbooks/bcpc/recipes/mysql.rb

```diff
@@ -120,12 +120,36 @@
 # 'service' command.
 #
 service 'mysql' do
-  action [:enable, :start]
+  action :enable
```

Comment:
Will the below code still start a MySQL daemon if it is simply down and the quorum is stable?

Reply:
If there are any live servers then just start this one
```
if status_collection.include? "ON" then
     self.notifies :start, 'service[mysql]', :immediate
   else
```

Else bootstrap
```self.notifies :run, 'execute[bootstrap MySQL quorum]', :immediate```

Bootstrap also involves a start
```
execute 'bootstrap MySQL quorum' do
  command 'service mysql bootstrap-pxc'
  action :nothing
end
```

Of course this means that if we have a quorum and it is currently down whilke adding a neew server to the forum we wind up bootstrapping a blank host and making it master, however `ruby_block 'decide to bootstrap MySQL quorum' do` only executes bootstrap if `node['bcpc']['mysql']['bootstrap_on_error'] == true ` and this is only true on a DEV cluster